src/globus_sdk/services/transfer/client.py

```diff
@@ -97,6 +98,7 @@ class TransferClient(client.BaseClient):
     """
     service_name = "transfer"
     base_path = "/v0.10/"
+    transport_class: Type[TransferRequestsTransport] = TransferRequestsTransport
```

Comment:
Is it necessary to annotate this with `Type[...]`? I would have thought that this wouldn't be needed, but maybe `mypy` can't understand without it.

Reply:
I just did what BaseClient did, was having trouble getting mypy to run locally for some reason

Reply:
Okay, if I remember I might play with this later.
(Aside: did `tox -e mypy` work for you? `mypy` with and without virtualenvs active gets tricky, so I'd rely on the tox target to setup a "good" install where libraries load correctly.)